eng/build.sh

```diff
@@ -125,6 +125,14 @@ while [[ $# > 0 ]]; do
       arguments="$arguments /p:BuildNativeStripSymbols=true"
       shift 1
       ;;
+     -coreclrConfiguration)
```

Comment:
This should be lowercase... I believe we keep lowercase because of case sensitivity in Unix. We did that for all arguments you can see subsetCategory, for example.